main/src/com/google/refine/expr/functions/strings/SplitByLengths.java

```diff
@@ -69,7 +69,7 @@ public Object call(Properties bindings, Object[] args) {
     
     @Override
     public String getDescription() {
-        return "Returns the array of strings obtained by splitting s into substrings with the given lengths. For example, \"internationalization\".splitByLengths(5, 6, 3) returns an array of 3 strings: [ \"inter\", \"nation\", \"ali\" ]. Excess characters are discarded.";
+        return "Returns the array of strings obtained by splitting s into substrings with the given lengths. For example, \"internationalization\".splitByLengths(5, 6, 3) returns an array of 3 strings: [ \"inter\", \"nation\", \"ali\" ]. Excess characters are discarded from the output array. \nLike other functions that returns an array, it also allows array slicing on the returned array. In that case, it returns the array consisting of a subset of elements between i1 and (i2 – 1). For example, (a) \"internationalization\".splitByLengths(5, 6, 3)[0,3] returns an array of 3 strings: [\"inter\", \"nation\", \"ali\"] \n (b) \"internationalization\".splitByLengths(5, 6, 3)[0,2] returns an array of 2 strings: [\"inter\", \"nation\"] \n (c) \"internationalization\".splitByLengths(5, 6, 3)[1,3] returns an array of 2 strings: [\"nation\", \"ali\" ] \n (d) \"internationalization\".splitByLengths(5, 6, 3)[1] returns string at position 1: \"nation\"";
```

Comment:
```suggestion
        return "Returns the array of strings obtained by splitting s into substrings with the given lengths. For example, \"internationalization\".splitByLengths(5, 6, 3) returns an array of 3 strings: [ \"inter\", \"nation\", \"ali\" ]. Excess characters are discarded from the output array.
```

This is way too much, and not specific, and in this small window, it's important to stay brief and on target. I wouldn't add this:
`Like other functions that returns an array, it also allows array slicing on the returned array. In that case, it returns the array consisting of a subset of elements between i1 and (i2 – 1). For example, (a) \"internationalization\".splitByLengths(5, 6, 3)[0,3] returns an array of 3 strings: [\"inter\", \"nation\", \"ali\"] \n (b) \"internationalization\".splitByLengths(5, 6, 3)[0,2] returns an array of 2 strings: [\"inter\", \"nation\"] \n (c) \"internationalization\".splitByLengths(5, 6, 3)[1,3] returns an array of 2 strings: [\"nation\", \"ali\" ] \n (d) \"internationalization\".splitByLengths(5, 6, 3)[1] returns string at position 1: \"nation\"";`